x-pack/plugins/enterprise_search/public/applications/esre/components/esre_guide/measure_performance_section.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiSteps, EuiText, EuiTitle } from '@elastic/eui';
+import { EuiContainedStepProps } from '@elastic/eui/src/components/steps/steps';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+const steps: EuiContainedStepProps[] = [
+  {
+    title: i18n.translate('xpack.enterpriseSearch.esre.measurePerformanceSection.step1.title', {
+      defaultMessage: 'Create a collection',
+    }),
+    children: (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.esre.measurePerformanceSection.step1.description"
+        defaultMessage="Visit Behavioral Analytics and create your first collection"
+      />
+    ),
+    status: 'incomplete',
+  },
+  {
+    title: i18n.translate('xpack.enterpriseSearch.esre.measurePerformanceSection.step2.title', {
+      defaultMessage: 'Integrate the analytics tracker',
+    }),
+    children: (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.esre.measurePerformanceSection.step2.description"
+        defaultMessage="After creating a collection, follow the directions on how to integrate our tracker into your application or website."
+      />
+    ),
+    status: 'incomplete',
+  },
+  {
+    title: i18n.translate('xpack.enterpriseSearch.esre.measurePerformanceSection.step3.title', {
+      defaultMessage: 'Review your dashboard',
+    }),
+    children: (
+      <FormattedMessage
+        id="xpack.enterpriseSearch.esre.measurePerformanceSection.step3.description"
+        defaultMessage="Our dashboards and tools help you visualize your end-user behavior and measure the performance of your search applications."
+      />
+    ),
+    status: 'incomplete',
+  },
+];
+
+export const MeasurePerformanceSection: React.FC = () => (
+  <>
+    <EuiFlexGroup direction="row" alignItems="center">
+      <EuiFlexItem grow={4}>
+        <EuiFlexGroup direction="column" gutterSize="s" justifyContent="flexStart">
+          <EuiFlexItem grow={false}>
+            <EuiTitle>
+              <p>
```

Comment:
A title should have an `h` child, not a `p` child--otherwise it's not a title. If you want to change the size without making it a title, you should use the size prop on `EuiTitle` or `EuiText`